Gems/Atom/Feature/Common/Assets/Passes/Forward.pass

```diff
@@ -92,9 +92,9 @@
                     "LoadStoreAction": {
                         "ClearValue": {
                             "Value": [
-                                0.4000000059604645,
-                                0.4000000059604645,
```

Comment:
Will that have any effect on the outcome of ASV for example? did the tests pass?

Reply:
Ah I bet it will, let me check! Will post those test results

Reply:
Yup causes the tests to fail - I'll change the code so it only updates the forward pass clearvalue when stars are enabled